### DISCUSSION TITLE
Filezilla

### DISCUSSION THREAD

**Discussion by aksaraicu** [pockethost/pockethost#193] [Q&A] [answered]
Hello, I'm trying to connect via FileZilla but it cannot retrieve the data. However, it displays a statement that I'm logged in. Is there any solution available?

"Error:	Disconnected from server: ECONNABORTED - Connection aborted
Error:	Failed to retrieve directory listing"

**Comment by benallfree** [comment, marked as answer]
@aksaraicu This may be a bug. #187 sounds similar.

**Reply by aksaraicu** [reply]
Hello, no worries, it's all done. And you know what? I made an AI websites using pocketbase! Here's a showcase https://aksara.icu

**Reply by benallfree** [reply]
Very nice! Please add the link to the #180 discussion :)
